.rubocop.yml

```diff
@@ -47,7 +47,7 @@ Metrics/ClassLength:
   Max: 200
 
 Metrics/ParameterLists:
-  Max: 10
+  Max: 15
```

Comment:
I am not sure it is a good sign, to be honest. Maybe it is time to switch to some `options`/`allowed_options` and something.